.github/workflows/test.yml

```diff
@@ -6,6 +6,22 @@ env:
   npm_config_cache: /tmp/npm-cache
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v4
+      - name: "Use Node.js"
+        uses: actions/setup-node@v4
+        with:
+          node-version: 23
+          cache: "npm"
```

Comment:
Apparently this causes the "Dependencies lock file is not found" error.Can you remove it, as well as the lockfile?

Given that `hafas-client` is a library (and not an app/service), the lockfile would only apply when cloning it and installing dependencies via `npm install`, so it has very limited benefits. At the same time, it is cumbersome to work with it. This is why I'd like the repo not to have a lockfile.

Reply:
> Apparently this causes the "Dependencies lock file is not found" error.

Ah makes sense :bulb: 

> Can you remove it, as well as the lockfile?

Yes and done :slightly_smiling_face: